packages/react-dom/src/client/ReactDOMSelection.js

```diff
@@ -150,11 +152,13 @@ export function getModernOffsetsFromPoints(
  * @param {object} offsets
  */
 export function setOffsets(node, offsets) {
-  if (!window.getSelection) {
+  const doc = node.ownerDocument || document;
+  const win = doc ? doc.defaultView : window;
+  if (!win.getSelection) {
```

Comment:
Just doing some poking around:

![image](https://user-images.githubusercontent.com/590904/42853894-c9957a7c-89fe-11e8-8895-d91ee485725c.png)

In what cases does `win` not have `getSelection`, and would it be good to document this?

Reply:
In the case of IE8, and possibly other very old browsers. We can remove the early return if that no longer matters.

Reply:
Cool. That is fine by me. Sound good, @aweary?